### ISSUE TITLE
[v5]Blacklist not used in getStoredState, breaking change from v4 not documented

### ISSUE THREAD

**Issue by sam-vdp** [rt2zz/redux-persist#518]
In v4, when a reducer was blacklisted from persistence, the reducer was a) never persisted and b) never rehydrated (see https://github.com/rt2zz/redux-persist/issues/107#issuecomment-222291475). However in v5, the blacklist has once more been removed from getStoredState. This is potentially a breaking change, if one relies on getStoredState using the blacklist.

**Comment by rt2zz** [comment]
v5 does things a bit different, the entire state get stored in one key, so doing a blacklist check on read doesnt do much. If you are getting blacklisted state it sounds like there may be a bug in the createPersistoid method.

Perhaps related to https://github.com/rt2zz/redux-persist/pull/521

**Comment by sam-vdp** [comment]
I am only getting blacklisted state if the blacklist was changed after the persist step. I was just slightly surprised by the change of behavior to v4 and thought maybe it should be mentioned in the docs, but it's probably too much of an implementation detail. Please feel free to close this.

**Comment by rt2zz** [comment]
Ok I think I better understand now there was a related issue that was recently fixed where we did not clear removed state: https://github.com/rt2zz/redux-persist/commit/e9549a8f3764b13b7acb8a7068396cf6df18abcd

Perhaps stateReconciler should additionally respect the blacklist during reconcilliation

**Comment by DavideDaniel** [comment]
Yeah.... this almost bit us with jwts getting stored... what needs to be done? Maybe we can move this along.
